jetbrains-core/src/software/aws/toolkits/jetbrains/core/credentials/ProjectAccountSettingsManager.kt

```diff
@@ -107,7 +110,9 @@ class DefaultProjectAccountSettingsManager(private val project: Project) :
         @Throws(CredentialProviderNotFound::class)
         get() = activeProfileInternal
             ?: tryOrNull {
-                getCredentialProviderOrNull("${ProfileToolkitCredentialsProviderFactory.TYPE}:default")
+                getCredentialProviderOrNull(ProfileToolkitCredentialsProviderFactory.DEFAULT_PROFILE_DISPLAY_NAME)
+            }?.takeIf {
```

Comment:
This will trigger MFA for loading of certain UIs that dont need it yet such as editing a run config

Reply:
Does my change trigger MFA where the existing code does not?